### ISSUE TITLE
Missing License Information

### ISSUE THREAD

**Issue by hellekin** [nQuire/sciencetoolkit#6]
Hi, I was directed to your nQuire "open source" project and I can't figure out the license.

First, there's no link to the source code from your website, and I found this repository by searching Github -- I'm happy to have found it!

Then, I looked into the Java source code but there's no mention of a software license.

There's no LICENSE file is any repository I looked into.

So, what's the license you're using? Did you decide yet? I would recommend AGPL-3 to cover network transfer and ensure the code remains in the public. Any EUPL-1.2-compatible license would do though.

**Comment by nfreear** [comment]
Hi @hellekin and @evilfer,

My quick research suggests that the `AGPL-3` license is most relevant for SAAS, which in this case would be the [nQuire-it](https://github.com/IET-OU/nquire-web-source) web-site and service itself.

As an Android app like _Sense-it_ is downloaded (_conveyed_) to and installed on the phone/ device, GPL-3.0+ seems fine.

Does this make sense?

Ta,

Nick

---
 * [Stackoverflow: 2127246/difference-between-affero-gpl-and-gplv3](https://stackoverflow.com/questions/2127246/difference-between-affero-gpl-and-gplv3)
 * [Stackexchange: 131264/what-constitutes-distributing-for-lgpl-v3](https://softwareengineering.stackexchange.com/questions/131264/what-constitutes-distributing-for-lgpl-v3)


Cc'ing :- @fisharebest

**Comment by evilfer** [comment]
Hi Nick,

I'm happy with those. The app doesn't use any code from the website, so
there are no issues with having different licenses. However we had decided
that the Sense-it *icons* would have a proprietary license, not allowing
reuse. If this hasn't changed, I suppose Sense-it will need to have two
licenses, one for the code, another for the icons.

Eloy

**Comment by fisharebest** [comment]
`AGPL-3.0-or-later` is preferable when you want the authors of derived versions to contribute their updates back to the original.

So, if we have any sort of long term plans for nQuire-IT, then this would be the better choice.

But if we subsequently decide that we don't have resources or inclination to continue the project, then there would be nothing to stop us from "downgrading" the licence at a later point, and re-publishing under `GPL-3.0-or-later` (or any other less-restrictive licence).

So, AGPL is probably the best choice for the nQuire website.

The app falls into the more traditional category of source/compiled code.  For this, the `GPL-3.0-or-later` is probably the most appropriate.

TL;DR - I agree with @nfreear  ;-)

**Comment by nfreear** [comment]
Thanks both.

FYI, from the '_About Sense-it_' page in the app ([nQuire/sciencetoolkit ../spotit/src/main/res/values/strings.xml](https://github.com/nQuire/sciencetoolkit/blob/master/spotit/src/main/res/values/strings.xml#L43-L69)):

> Developers: Mike Sharples, Eloy Villasclaras, Christothea Herodotou.
> This software is distributed under a GPLV3 licence.
> ...
> ...
> Copyright © 2014-17, <a href="http://www.open.ac.uk">The Open University</a>. All rights reserved.

Closing, N.
